backend/routes/room.js

```diff
@@ -8,21 +8,65 @@ const users = require('../models/User.model');
 const shortid = require('shortid');
 var io = require('../index');
 const auth = require('../middleware/auth');
-
+const cron = require('cron').CronJob;
 //Create a new room
 /* TODO: Modify the function rollback changes on failure*/
+
+//Run this function as a chron job every x hours
+function cleanDB() {
+  const now = new Date();
+  // console.log(now);
+  rooms.find({ lastreq: { $lt: now } }, (err, room) => {
```

Comment:
Why there's a need of condition of last req?

Anything is going to less them now whenever this function will be called.
Am i missing something ?

Reply:
lastreq is basically the time when any user in the room last joined the call. And then when cleaning is done it will take all the entries and then check one where lastreq is more than 12hrs and clear guests and online array

Reply:
I got that, im saying condition is not required
We can just do rooms.find({lastreq})

Ok but let it be for now wont affect the working